src/renderer_gl.cpp

```diff
@@ -6992,8 +6992,44 @@ namespace bgfx { namespace gl
 					, bx::uint32_imax(depth, 1)
 					) );
 			}
+			else
+			{
+				BX_ASSERT(depth <= 1, "Blitting 3D regions is not supported");
+
+				GLuint fbo;
+				GL_CHECK(glGenFramebuffers(1, &fbo) );
+
+				GL_CHECK(glBindFramebuffer(GL_FRAMEBUFFER, fbo) );
+
+				GL_CHECK(glFramebufferTexture2D(GL_FRAMEBUFFER
+						, GL_COLOR_ATTACHMENT0
+						, GL_TEXTURE_2D
+						, src.m_id
+						, bi.m_srcMip
+						) );
+
+				BX_ASSERT(GL_FRAMEBUFFER_COMPLETE == glCheckFramebufferStatus(GL_FRAMEBUFFER)
+					, "glCheckFramebufferStatus failed 0x%08x"
+					, glCheckFramebufferStatus(GL_FRAMEBUFFER)
+				);
+
+				GL_CHECK(glActiveTexture(GL_TEXTURE0) );
+				GL_CHECK(glBindTexture(GL_TEXTURE_2D, dst.m_id) );
+
+				GL_CHECK(glCopyTexSubImage2D(GL_TEXTURE_2D
+					, bi.m_dstMip
+					, bi.m_dstX
+					, bi.m_dstY
+					, bi.m_srcX
+					, bi.m_srcY
+					, width
+					, height) );
+
+				GL_CHECK(glBindFramebuffer(GL_FRAMEBUFFER, 0) );
+				GL_CHECK(glDeleteFramebuffers(1, &fbo) );
+			}
 		}
-	}
+   	}
```

Comment:
Would be better to have this on the left, aligned with its counterpart

Reply:
oh it is, but I accidently replaced the tab by 4 spaces on this lines (don't get me started about tabs in source code :))

Reply:
fixed it